go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/google/uuid v1.1.1
 	github.com/gophercloud/gophercloud v0.1.0 // indirect
 	github.com/gorhill/cronexpr v0.0.0-20180427100037-88b0669f7d75
+	github.com/grantae/certinfo v0.0.0-20170412194111-59d56a35515b // indirect
```

Comment:
i don't see grantae/certinfo imported anywhere in these changes -- just wondering where this indirect is needed